doc/sdk-cli-reference.md

```diff
@@ -460,7 +460,8 @@ Runs the tests locally
 * `--kubeconfig` string - location of kubeconfig for Kubernetes cluster (default "~/.kube/config")
 * `--global-manifest` string - path to manifest for global resources (default "deploy/crd.yaml)
 * `--namespaced-manifest` string - path to manifest for per-test, namespaced resources (default: combines deploy/service_account.yaml, deploy/rbac.yaml, and deploy/operator.yaml)
-* `--namespace` string - if non-empty, single namespace to run tests in (e.g. "operator-test") (default: "")
+* `--deploy-namespace` string - if non-empty, single namespace to run tests in (e.g. "operator-test") (default: "default")
```

Comment:
Is `"default"` the actual default? Isn't it an empty string, which results in a random namespace?

The changes in this file should match the help text exactly.

@theishshah @asmacdo Do we still need this file now that we're generating CLI docs?

Reply:
Agreed. Good catch.